pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/component/PirDetachedWebViewProvider.kt

```diff
@@ -46,7 +46,7 @@ interface PirDetachedWebViewProvider {
     ): WebView
 }
 
-@ContributesBinding(ServiceScope::class)
+@ContributesBinding(AppScope::class)
```

Comment:
I had to change most dependencies as the scheduled scans are not going to run on a service.

Reply:
Have to confirm though that ServiceScope will not work